pages/faq/index.public.js

```diff
@@ -25,6 +25,16 @@ Queremos ter conteúdo de qualidade tanto na publicação principal quanto nos c
       question: 'Que tipo de assunto é aceito no TabNews?',
       answer: `O conteúdo publicado no TabNews deve estar diretamente relacionado à tecnologia. Alguns exemplos de assuntos diretamente relacionados à tecnologia são: desenvolvimento de software, análise de dados, design, inteligência artificial, modelagem 3D, edição de vídeo, manipulação de imagens etc. Exemplos de assuntos indiretamente relacionados à tecnologia, mas que podem ser abordados do ponto de vista da tecnologia, são: produtividade, empreendedorismo, criação de conteúdo etc.`,
     },
+    {
+      id: 'qualidade-tabnews',
+      question: 'Como criar um bom conteúdo no TabNews?',
+      answer: `A qualidade é um conceito subjetivo, mas conteúdos de qualidade costumam ter características em comum. Pensando nisso, temos algumas recomendações que podem ajudar a criar um conteúdo com mais qualidade: 
```

Comment:
Será que qualidade é mesmo um "conceito subjetivo"? Acho que os critérios de qualidade de qualquer coisa podem ou não ser subjetivos, mas não o conceito de qualidade em si. É verdade que existem diferentes definições para qualidade, mas nenhuma me parece subjetiva.

Posso estar enganado quanto a isso, mas talvez seja melhor começar com algo como:

```suggestion
      answer: `A forma como cada pessoa avalia a qualidade de um conteúdo é subjetiva, mas temos algumas recomendações que podem ajudar a criar uma publicação mais relevante:
```

Reply:
Eu acredito que ficou melhor sim.

Minha primeira impressão foi que não é correto dizer que uma publicação X, quando seguindo as recomendações abaixo, seria _mais relevante_. Mas, decidi refletir a fundo sobre isso para conseguir dar um exemplo para ilustrar essa minha opinião. No fim, cheguei num pensamento de que tudo que pode ser publicado no TabNews tem o potencial de ser relevante, basta a pessoa ter cuidado o suficiente com o conteúdo.

Por exemplo, a publicação [Problema de eficiência energetica Linux VS Windows \[Mistério da Bateria\]](https://www.tabnews.com.br/Hanufu/problema-de-eficiencia-energetica-linux-vs-windows-misterio-da-bateria). Se o autor apenas criasse uma publicação dizendo que o computador dele gasta muita bateria no Window e pouca no Linux, provavelmente seria uma publicação ruim. Mas o autor foi além: adicionou bastante pesquisa, formatou bem a publicação, escreveu corretamente. Ou seja, virou um conteúdo bem melhor e acabou recebendo bastante atenção no dia, ficando entre os conteúdos mais relevantes.

Então, acho que faz sentido dizermos que essas características ajudam o conteúdo a ser mais relevante.